ci/tasks/upgrade-stemcell.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+set -e
+
+dir=`dirname "$0"`
+source ${dir}/utils.sh
+
+base=$( cd "$( dirname "$( dirname "$0" )")"/.. && pwd )
+base_gopath=$( cd $base/../../../.. && pwd )
+go version
+export GOPATH=$base_gopath:$GOPATH
+echo "GOPATH=" $GOPATH
+
+print_title "INSTALL BOSH CLI..."
+echo "installing bosh CLI"
+gem install bosh_cli --no-ri --no-rdo c
```

Comment:
Can we start preparing for the new BOSH CLI... maybe have the section that download CLI and issues CLI command be a separate script since new CLI will have different params and options. 

(not important but interesting to do if you have time)